File: core/src/main/java/foi/cverglici/core/data/model/student/nutritiongoal/NutritionGoalResponse.kt
package foi.cverglici.core.data.model.student.nutritiongoal

data class NutritionGoalResponse(
    val caloriesGoal: Int,
    val proteinsGoal: Double,
    val fatsGoal: Double,
    val carbohydratesGoal: Double,
    val goalSetDate: String
)

File: SmartMenza.Business/Models/NutritionGoals/NutritionGoalResponse.cs
namespace SmartMenza.Business.Models
{
    public class NutritionGoalResponse
    {
        public int CaloriesGoal { get; set; }
        public decimal ProteinsGoal { get; set; }
        public decimal FatsGoal { get; set; }
        public decimal CarbohydratesGoal { get; set; }
        public DateTime GoalSetDate { get; set; }
    }
}
